### PR TITLE
tag GeometryTypes, FixedSizeArrays and FreeType

### PR DESCRIPTION
**Pull request by SimonDanisch**
**GeometryTypes:**
+ polygon triangulation
+ removed depwarns
+ FlexibleSimplex, FlexibleConvexHull
+ bug fixes

**FixedSizeArrays:**
+ scalar isnan
+ fixes

**FreeType**
+ removed pointer argument restriction to let ccall do the proper conversion

### CODE REVIEW

File: FreeType/versions/1.1.0/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+@windows WinRPM
+Compat
```
Comment (tkelman) [inline FreeType/versions/1.1.0/requires]: should add a minimum version that implements the features being used

**Comment by tkelman** [comment]
what is the license status of https://github.com/JuliaGeometry/GeometryTypes.jl/pull/58, exactly?

also when you add features, you should really be incrementing minor version

**Comment by SimonDanisch** [comment]
I've asked the author and never got a reply :(

**Comment by tkelman** [comment]
then you don't have permission to license it under mit

**Comment by SimonDanisch** [comment]
Uh... so what are my options?

**Comment by Keno** [comment]
Send another email or write a clean-room re-implementation.

**Comment by SimonDanisch** [comment]
Okay, thanks!

**Comment by tkelman** [comment]
Where did it come from exactly? The site may have copyright info somewhere

**Comment by SimonDanisch** [comment]
It seems that the author has the same code here: http://wiki.ros.org/convex_decomposition
Which does use the MIT license, as far as I can tell.
(http://docs.ros.org/jade/api/convex_decomposition/html/triangulate_8cpp_source.html)

**Comment by tkelman** [comment]
good, hopefully that contribution was done by the author instead of copy-pasted by someone else. If you attribute that MIT license then you should be fine.

**Pull request closed without merging**